adds game command for setting game speed, and adds keyboard shortcuts (#1310)
OpenLoco/OpenLoco: CHANGELOG.md

- Feature: [#1278] Added a cheat to change the in-game date.
- Feature: [#1284] Added automatic loan repayment option in company finances window.
- Feature: [#1291] Modified the time panel date format to display day.

> - Feature: [#1310] Added keyboard shortcuts and a game command for game simulation speed.

Reviewer: We usually leave out the underlying technical explanations. Perhaps let's just go with:
```suggestion
- Feature: [#1310] Added keyboard shortcuts for changing the game simulation speed.
```
